packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -217,6 +217,42 @@ new ec2.Vpc(this, 'TheVPC', {
 provider.connections.allowFrom(ec2.Peer.ipv4('1.2.3.4/8'), ec2.Port.tcp(80));
 ```
 
+### Ip Address Management - IPAM
+
+To facilitate options for flexibility in the allocation and use of IP space within Vpc and Subnet including strategy and implementation for the allocation, Vpc utilizes an IpamProvider.
+
+By default, the Vpc construct will create and use a StaticIpam for you if you do not pass an `ipamProvider` to Vpc. StaticIpam is an implementation of the Original CDK Vpc and Subnet IP address allocation strategy and implementation. 
```

Comment:
This is all documentation very much in terms of code mechanism and artifacts. I'd like it  more in terms of the user experience and use cases. How about something like:

---------

The VPC spans a single large IP range, which contains the IP ranges of all its subnets. There are two possible sources for this IP range:

* You specify an IP range directly by specifying a CIDR 
* You allocate an IP range of a given size automatically from AWS IPAM

If you don't specify anything, by default the Vpc will allocate the `10.0.0.0/16` address range which will be exhaustively spread across all subnets in the subnet configuration. 

Be aware that the address space will be fully allocated! If you predict you may need to add more subnet groups later, add them early on and set `reserved: true` (see the "Advanced Subnet Configuration" section for more information).

## Specifying an IP range via a CIDR

...explanation and sample code here...

## Allocating an IP range from AWS IPAM

...explanation and sample code here...

Reply:
In the case where a VPC has not had an ipam pool assigned to it, its pretty clear that the intention is to not use one.  Your manually providing a cidr ( or relying on the default )..

It seems really confusing then to create a so called 'StaticIPAM',     the creation of the subnets is not an IPAM function.

Reply:
changes made in readme